Sources/Leaf/Node+Rendered.swift

```diff
@@ -8,7 +8,7 @@ extension Node {
         case let .number(number):
             return number.description.bytes
         case let .string(str):
-            return str.bytes
+            return str.htmlEscaped().bytes
```

Comment:
I think there needs to be an easier way for someone to do 

```
#(name) // Foo&lt;
#raw(name) // Foo<
```

In mustache this is the difference between `{{ name }}` and `{{{ name }}}`